HostedServices/ScheduledEventHostedService.cs

```diff
@@ -0,0 +1,78 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using snipetrain_bot.Services;
+
+namespace snipetrain_bot.HostedServices
+{
+    public class ScheduledPartyHostedService : IHostedService, IDisposable
+    {
+
+        private Timer _timer;
+        private readonly ILogger<ScheduledPartyHostedService> _logger;
+        private readonly IPartyService _partyService;
+        private readonly DiscordRunner _runner;
+
+        public ScheduledPartyHostedService(IPartyService partyService, DiscordRunner runner)
+        {
+            _partyService = partyService;
+            _runner = runner;
+        }
+
+        public Task StartAsync(CancellationToken stoppingToken)
+        {
+
+            var seconds = 60;
+            _timer = new Timer(DoWork, null, TimeSpan.Zero, TimeSpan.FromSeconds(seconds));
+
+            return Task.CompletedTask;
+        }
+
+        private async void DoWork(object state)
+        {
+            var partyList = await _partyService.GetPartiesAsync();
```

Comment:
I would wrap this code in a Try Catch as any exceptions thrown could cause this service to crash completely.